### DISCUSSION TITLE
Issue with YouTube Previews on Mastodon Instance After Migration

### DISCUSSION THREAD

**Discussion by zkreml** [mastodon/mastodon#31249] [Q&A] [answered]
Hello everyone, I'm experiencing issues with YouTube link previews on my Mastodon instance. The latest version is installed on the instance, and it has been running without problems for almost a year, but now the previews are not showing up. We recently moved the server, so I'm wondering if this could be related. Does anyone know which configuration settings I should check to resolve this issue? The problem with previews is only with YouTube links. Other link previews work fine. I would appreciate any advice. Thank you!

**Comment by mcg** [comment]
I also recently noticed this. Couple of more anecdotal points. The same post without a preview on my local instance would have the preview on the originating instance. Both instances appeared to be running the same versions of Mastodon. This also seems to be an issue only with “youtu.be”  short links.

**Comment by c960657** [comment, marked as answer]
There is some general problem with previews for YouTube videos, see #31308.
